.github/workflows/sync-repos.yml

```diff
@@ -27,14 +27,14 @@ jobs:
               changed_paths+="$path;"
             fi
           done
-          echo "::set-output name=changed_paths::$changed_paths"
```

Comment:
According to Github, set-output is deprecated and will be disabled soon, and advised to use GITHUB_ENV